<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateemployee</class>
 <widget class="QMainWindow" name="updateemployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Update Employee Data</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff</string>
    </property>
    <property name="text">
     <string>Update Employee Data </string>
    </property>
   </widget>
   <widget class="QPushButton" name="updatebtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>610</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(191, 191, 191);
	
	background-color: rgb(80, 80, 80);
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 85, 255);
}</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>831</width>
      <height>481</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>25</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>23</number>
     </property>
     <item row="7" column="1">
      <widget class="QLineEdit" name="empEmailtxt">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="empAD2lb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Employee Address 2:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="empAD2txt">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="empAD1lb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Employee Address 1:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="empEmaillb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Employee Email:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="empNametxt">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="empGenderlb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Employee Gender:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="empBDlb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Birth of Date:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="empIDlb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Employee ID:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="empDPlb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Employee Department:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="empNameld">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Employee Full Name:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="empBDtxt">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="empNumberlb">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Employee Mobile Number:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="empAD1txt">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="empNumbertxt">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="text">
        <string>Employee ID cannot be changed only for searching.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>6</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Orbitron</family>
            <pointsize>6</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #fff</string>
          </property>
          <property name="text">
           <string>Male</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Orbitron</family>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #fff</string>
          </property>
          <property name="text">
           <string>Female</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Orbitron</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff</string>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="cancelbtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>610</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(191, 191, 191);
	background-color: rgb(80, 80, 80);
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 85, 255);
}
</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updatebtn_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>1020</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(191, 191, 191);
	
	background-color: rgb(80, 80, 80);
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 85, 255);
}</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelbtn_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>1020</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: rgb(191, 191, 191);
	background-color: rgb(80, 80, 80);
}
QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 85, 255);
}
</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>comboBox_2</tabstop>
  <tabstop>empNametxt</tabstop>
  <tabstop>empBDtxt</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>empNumbertxt</tabstop>
  <tabstop>empEmailtxt</tabstop>
  <tabstop>empAD1txt</tabstop>
  <tabstop>empAD2txt</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>cancelbtn</tabstop>
  <tabstop>updatebtn</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>updatebtn_2</tabstop>
  <tabstop>cancelbtn_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelbtn</sender>
   <signal>clicked()</signal>
   <receiver>updateemployee</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>640</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
